EngineeringModel/ViewModels/Dialogs/EngineeringModelSetupDialogViewModel.cs

```diff
@@ -1,242 +1,154 @@
-﻿// -------------------------------------------------------------------------------------------------
+﻿// --------------------------------------------------------------------------------------------------------------------
```

Comment:
Please create/update the unittests to test all the new changes, since this is pretty drastic.